complex/ft_comp.c

```diff
@@ -118,53 +118,40 @@ int ft_bind_comp(struct fid_ep *ep, uint64_t flags)
 	return 0;
 }
 
-int ft_comp_rx(void)
+int ft_comp_x(struct fid_cq *cq, struct ft_xcontrol *ft_x, char *x_str, int nowait)
```

Comment:
A couple of minor requests:  using static char for x_str, and pass in the wait time directly, rather than using a boolean.